index.html

```diff
@@ -13,30 +13,33 @@
     <title>Waseem Qureshi</title>
 </head>
 <body>
-    <header class="header-menu">
-    <nav class="main-nav">
-        <div class="logo-menu">
-        <div class="name animate-logo"><h1 class="my-name">Waseem</h1></div>
-        <div><ul class="menu-box"><img class="hamburg" src="images/Normal Button/Tertiary/Icons/Icon - Menu.png" alt="menu"></ul></div>
-    </nav>
-    </div>
-    </header>
 
-    <!-- Desktop Header And Navigation -->
+    <!-- Desktop & Mobile Header And Navigation -->
 
-    <header class="desktop-header">
-    <div class="desktop-menu"
-    <div class="name-desktop "><h1 class="desktop-h1 animate-logo">Waseem Qureshi</h1></div>
-    <nav class="desktop-nav">
-        <ul class="desktop-list">
-            <li class="desktop-item"><a class="top-links" href="#"><img src="images/Union.png" alt="illustration"></a></li>
-            <li class="desktop-item"><a class="top-links" href="#">Contact</a></li>
-            <li class="desktop-item"><a class="top-links" href="#">About</a></li>
-            <li class="desktop-item"><a class="top-links" href="#">Portfolio</a></li>           
-        </ul>
-    </nav>
-    </div>
-    </header>
+    <header>
+        <nav class="navbar">
+            <a href=# class="nav-branding">Waseem Qureshi</a>
+            
+            <ul class="navMenu">
+                <li class="nav-item">
+                    <a href=#portfolio class="navLink">Portfolio</a>
+                </li>
+                <li class="underline"><hr></li>
+                <li class="nav-item">
+                    <a href=#about class="navLink">About</a>
+                </li>
+                <li class="underline"><hr></li>
+                <li class="nav-item">
+                    <a href=#contact class="navLink">Contact</a>
+                </li>
+                <li class="underline"><hr></li>
+            </ul>
+            <button class="hamburger">
+                <a class="hamburgerIcon" href="#"><img src="./images/Icon-Menu.svg" alt=""></a>
+                <a class="closedIcon" href="#"><img src="./images/Closed-Icon.png" alt=""></a>
+            </button>
```

Comment:
- [ ] You did a great job on the functionalities but you need to work on the design, Please reduce the left-mergin for the drop-down menu and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.

### Your design 
![image](https://user-images.githubusercontent.com/83803414/195173794-7b0d6154-8a66-478b-bacb-390a6373fbf7.png)

### Figma design 
![image](https://user-images.githubusercontent.com/83803414/195174767-f711bdc5-a077-420e-a295-1b81a91944ee.png)